modules/games.js

```diff
@@ -0,0 +1,318 @@
+          const tactics = result.body["tactics"];
+          const puzzle = result.body["puzzle_rush"]?.best;
+          const toTime = (time) => `<t:${time}:F>`;
+          const overall = daily?.record ? `Overall:\nWins: ${daily.record.win}\nLosses: ${daily.record.loss}\nDraws: ${daily.record.draw}\nTime Per Move: ${daily.record['time_per_move']}\n\n` : "";
+          const latest = daily?.last ? `Latest:\nRating: ${daily.last.rating}\nDate: ${toTime(daily.last.date)}\n\n` : "";
+          const best = daily?.best ? `Best:\nRating: ${daily.best.rating}\nDate: ${toTime(daily.best.date)}\n[Link](${daily.best.game})` : "";
+          embed.setTitle(`Chess.com Stats for ${name}`)
+            .addField("Chess Daily", (overall || latest || best) ? `${overall}${latest}${best}` : "No available stats")
+            .addField("Puzzle Rush", puzzle ? `Total Attempts: ${puzzle["total_attempts"]}\nHigh Score: ${puzzle.score}` : "No available stats")
+            .addField("Tactics", tactics ? `Highest Rating: ${tactics.highest.rating}\nLowest Rating: ${tactics.lowest.rating}` : "No available stats");
+          return inter.reply({ embeds: [embed] });
+        }
+      }
+      if (games.length == 0) embed.setDescription(`No active games found for ${name}`);
+      if (games.length > 25) embed.setDescription(`${name}'s first 25 active games:`);
+      else embed.setDescription(`${name}'s active games:`);
+      inter.reply({ embeds: [embed] });
+    } catch (error) {
+      if (error.message == "Not Found" && error.statusCode == 404) {
+        inter.reply({ content: `I couldn't find a profile for \`${name}\`.`, ephemeral: true });
+      } else { u.errorHandler(error, inter); }
+    }
+  } else {
+    inter.reply({ content: "I couldn't find a saved IGN for them.", ephemeral: true });
+  }
+}
+
+/** @param {discord.CommandInteraction} inter */
+async function elite(inter) {
+  const starSystem = inter.options.getString('system-name') ? await eliteAPI.getSystemInfo(inter.options.getString('system-name')) : null;
+  const embed = u.embed().setThumbnail("https://i.imgur.com/Ud8MOzY.png").setAuthor({ name: "EDSM", iconURL: "https://i.imgur.com/4NsBfKl.png" });
+  const getStatus = async () => {
+    const status = await eliteAPI.getEliteStatus();
+    return inter.reply({ content: `The Elite: Dangerous servers are ${status.type == 'success' ? "online" : "offline"}`, ephemeral: true });
+  };
+  const getTime = async () => {
+    const d = new Date();
+    const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    return inter.reply(`The current date/time in Elite is ${monthNames[d.getUTCMonth()]} ${d.getUTCDate()}, ${(d.getUTCFullYear() + 1286)}, ${d.getUTCHours()}:${d.getUTCMinutes()}. (UTC + 1286 years)`);
+  };
+  const getSystem = async () => {
+    if (!starSystem) return inter.reply({ content: "I couldn't find a system with that name.", ephemeral: true });
```

Comment:
If no system is given, the message should direct the user to fill in the `system-name` parameter; this message may create confusion if they don't notice that `system-name` is a thing.